Sources/KSCrashFilters/include/KSCrashReportFilterAppleFmt.h

```diff
@@ -107,7 +107,8 @@ typedef NS_ENUM(NSInteger, KSAppleReportStyle) {
 NS_SWIFT_NAME(CrashReportFilterAppleFmt)
 @interface KSCrashReportFilterAppleFmt : NSObject <KSCrashReportFilter>
 
-+ (instancetype)filterWithReportStyle:(KSAppleReportStyle)reportStyle;
+- (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
Default constructor makes sense here IMO

Reply:
What type do you think should be the default?

Reply:
Implicit 0 because this class doesn't have ivar initialization and relies on class memory initialization with 0.